src/components/LenderProfile/LenderMap.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+	<section>
+		<h4 class="data-hj-suppress tw-mb-2">
+			{{ lenderMapTitle }}
+		</h4>
+		<kv-map
+			class="
+				tw-rounded
+				tw-overflow-hidden
+				tw-mb-3
+			"
+			:auto-zoom-delay="500"
+			:aspect-ratio="1.8"
+			:lat="30"
+			:long="1"
+			:zoom-level="2"
+			:use-leaflet="true"
+			:show-zoom-control="true"
+			:allow-dragging="true"
+			:show-labels="false"
+			:countries-data="countriesData"
+			:show-fundraising-loans="showFundraisingLoans"
+		/>
+		<div>
+			<p class="tw-mb-1 tw-font-medium">
+				Legend:
+			</p>
+			<div class="tw-grid tw-grid-cols-2 md:tw-grid-cols-3 tw-gap-2">
+				<div
+					v-for="legend in legendData"
+					:key="legend.color"
+					class="tw-flex tw-items-start tw-gap-1"
+				>
+					<span
+						:class="[
+							'tw-w-4',
+							'tw-h-2',
+							'tw-border',
+							legend.color,
+						]"
+					></span>
+					<p class="tw-text-small tw-font-medium">
+						{{ legend.label }}
+					</p>
+				</div>
+			</div>
+			<div class="tw-flex tw-items-center tw-gap-1 tw-mt-2">
+				<span
+					class="tw-w-1 tw-h-1 tw-border tw-rounded-full tw-bg-brand-900"
+				></span>
+				<p class="tw-text-small tw-font-medium">
+					Fundraising loans
+				</p>
+			</div>
+		</div>
+		<div class="tw-mt-3">
+			<p class="tw-mb-1 tw-font-medium">
+				Options:
+			</p>
+			<kv-checkbox
+				class="tw-text-small tw-font-medium custom-checkbox"
+				v-model="showFundraisingLoans"
+			>
+				Show fundraising loans
+			</kv-checkbox>
+		</div>
+	</section>
+</template>
+
+<script>
+import numeral from 'numeral';
+import KvMap from '@/components/Kv/KvMap';
+import KvCheckbox from '~/@kiva/kv-components/vue/KvCheckbox';
+
+export const getIntervals = (min, max, nbIntervals) => {
```

Comment:
It would be worth adding a ticket to the epic to extract this to a util and unit test it.

Reply:
Also, what specifically are the intervals here? It would probably be good to add a jsdoc comment to this method to explain what it's generating.